docs/reference/platforms/windows/app.rst

```diff
@@ -14,6 +14,9 @@ enabled. To ensure .NET Framework 3.5 is enabled:
 3. Select "Turn Windows features On or Off"
 4. Ensure that ".NET framework 3.5 (includes .NET 2.0 and 3.0)" is selected.
 
+Instead of building an MSI installer, you can also pack the folders for
```

Comment:
This should be incorporated into the earlier paragraph introducing WiX; something like:

```
Briefcase supports 2 packaging formats for a Windows app:

1. As an MSI installer
2. As a Zipfile containing all files needed to run the app
```
then lead into the description of how WiX is used to build MSIs.

A similar comment/update is needed on the VisualStudio format as well.